index.js

```diff
@@ -1 +1,4 @@
-module.exports = {};
+import DockerLauncher from './src/launcher';
+
+export default DockerLauncher;
+export const launcher = DockerLauncher;
```

Comment:
New line?

Reply:
Done, btw, for wdio5 to load `wdio-docker-service`, it needs for it to export the `launcher` property (other than default).